Source/Core/Matrix3.js

```diff
@@ -1355,6 +1355,16 @@ define([
                                                 0.0, 0.0, 1.0));
 
     /**
+     * An immutable Matrix3 instance initialized to the zero matrix.
+     *
+     * @type {Matrix3}
+     * @constant
+     */
+    Matrix3.ZERO = freezeObject(new Matrix3(0.0, 0.0, 0.0,
```

Comment:
Where in CHANGES.md should I put it? Since I'm assuming the 1.10 section is final.